include/vkgcDefs.h

```diff
@@ -82,6 +82,7 @@
 //  %Version History
 //  | %Version | Change Description                                                                                    |
 //  | -------- | ------------------------------------------------------------------------------------------------------|
+//  |     54.6 | Add reverseThreadGroup to PipelineOptions                                                             |
```

Comment:
Make sure to sync the minor version. Right now there's a gap.

Reply:
There are some internal changes occupying those versions.